geps/gep-713/index.md

```diff
@@ -1,1157 +1,985 @@
-  Namespaces in the future
-* Provide a means of attachment that works for both ingress and mesh
-  implementations of this API
-* Provide a consistent specification that will ensure familiarity between both
-  included and implementation-specific policies so they can both be interpreted
-  the same way.
-
-## Deferred Goals and Discussions
-
-* Should Policy objects be able to target more than one object? At the time of
-  writing, the answer to this is _no_, in the interests of managing complexity
-  in one change. But this rule can and should be discussed and reexamined in
-  light of community feedback that users _really_ want this. Any discussion will
-  need to consider the complexity tradeoffs here.
+* Establish a pattern for metaresources which will be used for any metaresources and policies included in the Gateway API spec.
+* Establish a pattern that must be adopted for any implementation-specific metaresources and policies used with Gateway API resources.
+* Discuss the problems with communicating status for metaresource objects, and suggest mechanisms that APIs can use to mitigate some of them.
+* Provide a way to distinguish between required and default values for all metaresource API implementations.
+* Enable Policy Attachment at all relevant scopes in Gateway API, including Gateways, Routes, Backends, along with how values should flow across a hierarchy if necessary.
+* Ensure the Policy Attachment specification is generic and forward thinking enough that it could be easily adapted to other grouping mechanisms like Namespaces in the future.
+* Provide a means of attachment that works for both ingress and mesh implementations of Gateway API.
+* Provide a consistent specification that will ensure familiarity between both included and implementation-specific metaresources so they can both be interpreted the same way.
+* Provide a reference pattern to other implementations of metaresource and policy APIs outside of Gateway API, that are based on similar concepts (i.e., augmenting the behavior of other Kubernetes objects, attachment points, nested contexts and inheritance, Defaults & Overrides, etc.)
 
 ## Out of scope
 
-* Define all potential policies that may be attached to resources
-* Design the full structure and configuration of policies
+* Define all potential metaresource and policy kinds that may be attached to resources.
+
+## Background
+
+When designing Gateway API, it was found a frequent need to change the behavior of objects without being able to make changes to the spec of those objects. Sometimes, this is because changing the spec of the object to hold the new piece of information is not possible (e.g., `ReferenceGrant`, from [GEP-709](https://gateway-api.sigs.k8s.io/geps/gep-709/), when affecting Secrets and Services), and sometimes it’s because the behavior change is intended to flow across multiple objects (see [Semantics](#semantics-why) of metaresources and [Inherited](#inherited) class of metaresources).
```

Comment:
Should be either "we found" or "it was found that". "We found" is better because passive voice is harder to read.

Reply:
Rephrased. Thanks!